lua/telescope/pickers/layout_strategies.lua

```diff
@@ -599,16 +600,16 @@ layout_strategies.cursor = make_documented_layout(
       results.width = prompt.width
     end
 
-    local position = vim.api.nvim_win_get_position(0)
+    local position = vim.api.nvim_win_get_position(winid)
     local winbar = (function()
       if vim.fn.exists "&winbar" == 1 then
         return vim.o.winbar == "" and 0 or 1
```

Comment:
```suggestion
        return vim.wo[winid].winbar == "" and 0 or 1
```

i think we can change this as well.

Reply:
@jamestrew Good catch, I've added this to the changes.